examples/whisper/prepare_whisper_configs.py

```diff
@@ -74,10 +71,6 @@ def main(raw_args=None):
         "whisper_audio_decoder_dataloader" if not args.no_audio_decoder else "whisper_no_audio_decoder_dataloader"
     )
 
-    # update model specific values for transformer optimization pass
-    template_json["passes"]["transformers_optimization"]["config"]["num_heads"] = whisper_config.encoder_attention_heads
-    template_json["passes"]["transformers_optimization"]["config"]["hidden_size"] = whisper_config.d_model
```

Comment:
do we need remove the settings for whisper?

Reply:
Yes, currently, we can parse this values from model configs.